### PR TITLE
User production to run servers (flask and react)

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

**Comment by redhat-qe-bot1** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
 * To assign reviewers based on OWNERS file use `/assign-reviewers`
 * To check if PR can be merged use `/check-can-merge`

<details>
<summary>Supported /retest check runs</summary>

 * `/retest build-container`: Retest build-container

</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

**Comment by myakove** [comment]
pre-commit.ci run

File: api/api.py [hunk 1]
```diff
@@ -17,39 +13,34 @@
...
         db.session.commit()
 
 
-with app.app_context():
-    db.drop_all()
-    db.session.commit()
-    db.create_all()
-    create_users()
-    db.session.commit()
+# with app.app_context():
```
Comment (rnetser) [inline api/api.py:30]: leftovers?

File: api/utils.py [hunk 2]
```diff
@@ -125,47 +132,49 @@ def delete_all_claims(user: str) -> Dict[str, List[str]]:
...
+    return {"creds": f"Username {_secret.instance.data.username}:Password {_secret.instance.data.password}"}
 
 
-def get_claimed_cluster_kubeconfig(claim_name: str) -> str:
-    _cluster_deployment = get_claimed_cluster_deployment(claim_name=claim_name)
+def get_claimed_cluster_kubeconfig(claim_name: str, dyn_client: DynamicClient) -> Dict[str, str]:
+    _cluster_deployment = get_claimed_cluster_deployment(claim_name=claim_name, dyn_client=dyn_client)
     if isinstance(_cluster_deployment, str):
-        return ""
+        return {"kubeoncig": ""}
```
Comment (rnetser) [inline api/utils.py:169]:
```suggestion
        return {"kubeconfig": ""}
```

**Review by rnetser — CHANGES_REQUESTED** [review]

**Review by rnetser — APPROVED** [review]

**Comment by msi-b1** [comment]
New container for quay.io/redhat_msi/hive-claim-manager:latest published

**Pull request merged**